.github/CONTRIBUTING.md

```diff
@@ -274,7 +274,13 @@ Math operators like +, -, /, *, etc are up in the air, just choose which version
 #### Use
 * Bitwise AND - '&'
 	* Should be written as ```bitfield & bitflag``` NEVER ```bitflag & bitfield```, both are valid, but the latter is confusing and nonstandard.
-
+* Do not explicitly assign `null` to newly declared variables as this is already their default value
+* When defining new datums, place newly defined variables at the bottom
+* If a datum has already been declared somewhere, additional var definitions should not be made outside of that block
+* Associated lists declarations must have their key value quoted if it's a string
+	* WRONG: list(a = "b")
+	* RIGHT: list("a" = "b")
+* Do not call procs for reasons other than their intended purpose. (i.e. New(), Destroy(), Crossed(), Topic(), attackby(), attack_hand() should not be called directly). Create seperate procs that encapsule the functionality you're trying to perform instead.
```

Comment:
I'll argue that there's always 2 better ways to to it

Case 1: You want some sort of special effect from attack* to trigger. In that case it's better to have a common proc and call it from inside attackby and the place you were going to call attackby.

Case 2: You actually want to trigger an attack on the item. For that we have melee_item_attack_chain() which handles the process correctly.

Considering I want to start enforcing attackby() and attack_hand() start `. = ..()`ing (because there's some stuff that needs to go in the parent proc) having direct calls like these can lead to unintended side effects